docs/content/deployment/guides/celery.mdx

```diff
@@ -36,40 +54,42 @@ def parallel_job():
+You'll also need to run a Celery worker to execute tasks. From the same directory where you saved the `celery_job.py` file, run:
 
 ```shell
 dagster-celery worker start -A dagster_celery.app
 ```
 
-Now you can execute this job with Celery. Again, from the same directory in which you saved the job file, run:
+Next, execute this job with Celery by running the following:
 
 ```shell
 dagster dev -f celery_job.py
 ```
 
-Now you can execute the parallel job from the Dagster UI.
+Now you can execute the parallel job from the [Dagster UI](/concepts/webserver/ui).
+
+---
 
-## Ensuring workers are in sync
+## Part 2: Ensuring workers are in sync
 
-In the quick start, we cheated in a couple of ways:
+In Part 1, we took a few shortcuts:
 
-- By running a single Celery worker on the same node as the Dagster webserver, so that local ephemeral run storage and event log storage could be shared between them, and so that filesystem io manager would be sufficient to exchange values between the worker's task executions.
-- By running the Celery worker in the same directory to which we saved `celery_job.py`, so that our Dagster code would be available both to the webserver and to the worker: i.e., the worker and the webserver can both find the job definition in the same file (`-f celery_job.py`).
+- **We ran a single Celery worker on the same node as the Dagster webserver.** This allowed us to share local ephemeral run storage and event log storage between them and use the filesystem I/O manager to exchange values between the worker's task executions.
+- **We ran the Celery worker in the same directory as `celery_job.py`**. This meant that our Dagster code was available to both the webserver and the worker, specifically that they could both find the job definition in the same file (`-f celery_job.py`)
 
 In production, more configuration is required.
 
-### 1. Persistent run and event log storage
+### Step 1: Configure persistent run and event log storage
+
+First, you'll ensure that appropriate persistent run and event log storage, e.g., `PostgresRunStorage` and `PostgresEventLogStorage`, are configured on your [Dagster instance](/deployment/dagster-instance) (via `dagster.yaml`). This allows the webserver and workers to communicate information about the run and events with each other.
 
-First, ensure that appropriate persistent run and event log storage, e.g., **`PostgresRunStorage`** and **`PostgresEventLogStorage`**, are configured on your Dagster instance, so that the webserver and the workers can communicate information about the run and events. You can do this by adding a block such as the following to your `dagster.yaml` (by default, Dagster will look for this file at `$DAGSTER_HOME/dagster.yaml` when the `DAGSTER_HOME` environment variable is set):
+Add a block like the following to your project's `dagster.yaml`:
```

Comment:
can leave this but ideally we could just link to a page about configuring postgres storage